src/main/histFileMod.F90

```diff
@@ -3397,18 +3422,23 @@ subroutine htape_timeconst(t, mode)
        call ncd_putatt(nfid(t), ncd_global, 'time_period_freq',          &
                           trim(time_period_freq))
 
+       long_name = 'current seconds of current date at end of ' // step_or_bounds
        call ncd_defvar(nfid(t) , 'mcsec' , ncd_int, 1, dim1id , varid, &
-          long_name = 'current seconds of current date', units='s')
+          long_name = long_name, units='s')
+       long_name = 'current day (from base day) at end of ' // step_or_bounds
        call ncd_defvar(nfid(t) , 'mdcur' , ncd_int, 1, dim1id , varid, &
-          long_name = 'current day (from base day)')
+          long_name = long_name)
+       long_name = 'current seconds of current day at end of ' // step_or_bounds
        call ncd_defvar(nfid(t) , 'mscur' , ncd_int, 1, dim1id , varid, &
-          long_name = 'current seconds of current day')
+          long_name = long_name)
        call ncd_defvar(nfid(t) , 'nstep' , ncd_int, 1, dim1id , varid, &
           long_name = 'time step')
 
        dim2id(1) = hist_interval_dimid;  dim2id(2) = time_dimid
-       call ncd_defvar(nfid(t), 'time_bounds', ncd_double, 2, dim2id, varid, &
-          long_name = 'history time interval endpoints')
+       if (tape(t)%hlist(1)%avgflag /= 'I') then  ! NOT instantaneous fields tape
```

Comment:
Might be prudent to introduce a variable like `is_inst_tape` instead of repeating this conditional everywhere. That way, once you start segregating instantaneous/non variables onto their own tapes, you only have to change it in one place.

Reply:
Thank you @samsrabin 
I linked your comment to #2445 and will address it no later than at that time.

Reply:
The alternate if-statement [discussed below ](https://github.com/ESCOMP/CTSM/pull/2838#issuecomment-2492286937)resolves this.

In the context of treating instantaneous tapes differently with this PR and #2445, @samsrabin asked whether we loop through all of a tape's fields to identify instantaneous tapes at runtime. The answer is no, and we agreed that #2445 will handle this by separating all instantaneous fields onto separate tapes and, therefore, no need to worry about doing this now.